src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/package.json

```diff
@@ -3,9 +3,15 @@
   "displayName": "Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension",
   "description": "A companion extnesion for debugging Blazor WebAssembly applications in VS Code. Must be installed alongside the C# extension.",
   "version": "0.0.1",
+  "repository": {
+    "type" : "git",
+    "url" : "https://github.com/dotnet/aspnetcore-tooling.git"
+  },
+  "license": "Apache-2.0",
   "engines": {
     "vscode": "^1.52.0"
   },
+  "icon": "images/blazorIcon.png",
```

Comment:
You'll probably have to get verification from Marketing that this is OK to use. Beth Massi was the vscode-dotnet-runtime contact.

Reply:
Confirmed with @danroth27 for this. This is the same icon asset that we use elsewhere for Blazor so it should be fine.